common/persistence/cassandra/cassandraHistoryPersistence.go

```diff
@@ -356,17 +356,17 @@ func (h *cassandraHistoryV2Persistence) DeleteHistoryBranch(
 	for _, b := range rsp.Branches {
 		for _, br := range b.Ancestors {
 			// These string casts are safe and optimized away -- https://github.com/golang/go/issues/3512
```

Comment:
Is this comment on line 358 still applicable?

Reply:
i will update the comments in new PRs
```go
GetBranchId() string
```